bin/lp-builder.js

```diff
@@ -22,12 +22,13 @@ var config = {
   LP_VERSION: null,
   LP_APPS: null,
   LP_TASKS: ['copy', 'appmanifests', 'optimize'],
+  LP_GLOBAL_TASKS: ['copySpeechData'],
```

Comment:
Each `LP_TASKS` element is executed once for each app/locale combination, see `LangpackBuilder.prototype.build`. However, each `LP_GLOBAL_TASKS` element is executed once per locale.